Enable/View Controllers/MainNavigationViewController.m

```diff
@@ -0,0 +1,30 @@
+//
+//  MainNavigationViewController.m
+//  Enable
+//
+//  Created by Amanda Wang on 7/29/22.
+//
+
+#import "MainNavigationViewController.h"
+#import "ThemeTracker.h"
+@interface MainNavigationViewController ()
+
+@end
+
+@implementation MainNavigationViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+}
+
+- (UIStatusBarStyle)preferredStatusBarStyle{
+    NSString * status = [ThemeTracker sharedTheme].colorSet[@"StatusBar"];
+    if([status isEqualToString:@"Default"]){
+        return UIStatusBarStyleDefault;
+    } else if ([status isEqualToString:@"Light"]){
+        return UIStatusBarStyleLightContent;
+    }
+    return UIStatusBarStyleDarkContent;
```

Comment:
I'm nit picking, but I feel like it would make more sense to have "Dark" and "Light" in the theme rather than "Default" and "Light", just in case default become something else in the future, it would not match the theme anymore.
```
NSString * statusBar = [ThemeTracker sharedTheme].colorSet[@"StatusBar"];
switch(statusBar) {
   case 'Dark' :
      return UIStatusBarStyleDarkContent;
   case 'Light' :
     return UIStatusBarStyleLightContent;
   default :
      return UIStatusBarStyleDefault;
   }
```